### ISSUE TITLE
Origin about this repository

### ISSUE THREAD

**Issue by matiasbeckerle** [MakingSense/DTRU#1]
Hey @mravinale, @AlphaGit. Are there any reasons to keep this repository open? It looks empty.

**Comment by AlphaGit** [comment]
Not that I'm aware of. -- I propose deleting it.

**Comment by matiasbeckerle** [comment]
Thank you @AlphaGit.

@andresmoschini or @afantini any ideas? Someone to ping about it?

**Comment by andresmoschini** [comment]
Yes, delete it, I do not remember why we created it, but it was a lot time ago, and it is still empty. Thanks!

**Comment by matiasbeckerle** [comment]
Thanks @andresmoschini. Deleting...
